rossanafmenezes/Kakelandet_My_Cookbook
# Add navigation component

Closes: #9 


### Description


This PR adds:

1.  the navigation component and safeArgs dependencies
2. UI Files for all 5 screens
3. [Options Menu ](https://developer.android.com/develop/ui/views/components/menus#options-menu) to navigate between fragments (this is on the MVP only, in the future the app will have a bottom nav bar)

### Testing instructions


1. Load the app 
2. Click on the drawer menu
3. make sure each item navigates to the correct fragment

### Images/gif


<img src="https://user-images.githubusercontent.com/30724184/205456481-5490ba93-d041-487b-ab99-ba678f8d825b.gif" width="350"/>
